src/test/scala/is/hail/io/IndexBTreeSuite.scala

```diff
@@ -102,4 +102,29 @@ class IndexBTreeSuite extends SparkSuite {
       IndexBTree.write(index, idxFile, sc.hadoopConfiguration)
     }
   }
-}
\ No newline at end of file
+
+  @Test def testMultipleOfBranchingFactorDoesNotAddUnnecessaryElements() {
```

Comment:
Can you reformat this file? Also, this is a test that has a ridiculously long runtime. Now that you parameterized the branching factor, can you change the branching factor above? If you don't have time, I can do that after this PR goes in.

Reply:
Point-wise:

1. Yes.
2. 🤷‍♀️ it's a test name so it seems more OK since no one ever has to call it? and it's vaguely helpful to have descriptive names in the test report
3. "Change the branching factor above" meaning add an extra random parameter for branching factor?

Reply:
Sorry -- I wasn't clear. I'll make the change I was proposing separately.